src/ucp/core/ucp_mm.c

```diff
@@ -462,7 +462,7 @@ static void ucp_memh_cleanup(ucp_context_h context, ucp_mem_h memh)
 }
 
 static ucs_status_t ucp_memh_register_gva(ucp_context_h context, ucp_mem_h memh,
-                                          ucp_md_map_t md_map)
+                                          ucp_md_map_t md_map, unsigned uct_flags)
```

Comment:
I understand it's temp, but, just in case, it would be good to replace gva_mr with some structure and keep registrations for different access flags there.